modules/two_factor_authentication/app/views/two_factor_authentication/settings.html.erb

```diff
@@ -27,57 +27,60 @@
       <%= render(AttributeGroups::AttributeGroupComponent.new) do |component|
         component.with_attribute(key: t('two_factor_authentication.settings.label_active_strategies'),
                                  value: if configuration['active_strategies'].empty?
-                                          t(:label_none)
-                                        else
-                                          strategies = configuration['active_strategies'].map do |key|
-                                            content_tag(:span) do
-                                              t("two_factor_authentication.strategies.#{key}")
-                                            end
-                                          end
-                                          safe_join(strategies, tag(:br))
-                                        end)
+                                   t(:label_none)
+                                 else
+                                   strategies = configuration['active_strategies'].map do |key|
+                                     content_tag(:span) do
+                                       t("two_factor_authentication.strategies.#{key}")
+                                     end
+                                   end
+                                   safe_join(strategies, tag(:br))
+                                 end)
 
         component.with_attribute(key: t('two_factor_authentication.settings.label_enforced'),
                                  value: !!configuration['enforced'])
```

Comment:
This is shown as empty string when enforced set to false, probably better to show `false`